delegation/src/main/java/com/iluwatar/delegation/simple/App.java

```diff
@@ -0,0 +1,33 @@
+package com.iluwatar.delegation.simple;
+
+import com.iluwatar.delegation.simple.printers.CanonPrinter;
+import com.iluwatar.delegation.simple.printers.EpsonPrinter;
+import com.iluwatar.delegation.simple.printers.HpPrinter;
+
+/**
+ * In this example the delegates are {@link EpsonPrinter}, {@link HpPrinter} and {@link CanonPrinter} they all implement
+ * {@link Printer}. The {@link AbstractPrinterController} and through inheritance {@link PrinterController} also
+ * implement {@link Printer}. However neither provide the functionality of {@link Printer} by printing to the screen,
+ * they actually call upon the instance of {@link Printer} that they were instantiated with. Therefore delegating the
+ * behaviour to another class.
+ */
```

Comment:
Please also explain generally what delegation means. I think this should be at the top.